email-templates/award-badge-en_US/dojos-award-badge.html.ejs

```diff
@@ -0,0 +1,13 @@
+<% include ../common-en_US/header.ejs %>
+
+<h1>Hi <%=recipientName%></h1>
+
+<p>Congratulations! You have just been awarded an official <%=badgeName%> for your skills!
```

Comment:
We were talking before about this being `<%=badgeName%> badge`, since most badges don't have the word badge in them. Plus @petecoderdojo is redoing the badges, so we can get rid of "Badge" from those that do have that word in them.